### ISSUE TITLE
Suggestion: Increase priority for the Main MiSTer process

### ISSUE THREAD

**Issue by bootsector** [MiSTer-devel/Linux_Image_creator_MiSTer#4]
/etc/inittab entry for MiSTer process should be:

`::sysinit:nice -n -20 /media/fat/MiSTer &`

**Comment by sorgelig** [comment]
which function improved by this?

**Comment by bootsector** [comment]
I believe this process should be prioritized by the Linux kernel against all the other processes running in the system. Today, its priority is the same as all the other running processes. This configuration should put Main MiSTer in front of the others in the Kernel process scheduling, improving overall responsiveness (although not easily perceivable) of the services provided by Main MiSTer.

**Comment by sorgelig** [comment]
basically MiSTer process is running on separate CPU core, so it has 99.9% of time.

**Comment by bootsector** [comment]
That helps. But since it has same priority (20) as the other processes, kernel won't give it special treatment/priority if needed, for some reason.

**Comment by hmusta** [comment]
Yes, but those other Linux processes should ideally be essential for maintaining the system, so it's best that they get priority. The proper solution would be to identify processes related to non-essential services and just remove them if possible.

**Comment by bootsector** [comment]
Well, my point is: FTP server can wait, let's say, 16ms for CPU attention, but input processing inside Main MiSTer can't.

Can't see any "unnecessary" from htop here, but I see that, given the criticality of Main MiSTer, it should be given at least higher priority among other processes.

**Comment by sorgelig** [comment]
i think it's more about lowering the priority for less important tasks like FTP, than boosting the MiSTer.
